marketplace/deployer_helm_tiller_base/test/application.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.template_mode }}
```

Comment:
Instead of an implicit value like this, we should introduce a new field in the schema. This is required by the helm-tiller deployer, but not by the UI:

```
properties:
  existingApplicationUid:
    type: string
    x-google-marketplace:
      type: EXISTING_APPLICATION_UID
```

The deployer will populate this with the right UID it finds. This line in the template should be rewritted as:

```
{{- if not .Values.existingApplicationUid }}
```

Reply:
Created https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/pull/224.

Will update when merged.